Source/Mods/SimpleSidearms.cs

```diff
@@ -1,6 +1,7 @@
 ﻿using System;
 
 using HarmonyLib;
+using JetBrains.Annotations;
```

Comment:
How's JetBrains.Annotations sneaking in? Don't you need it in the csproj?

Reply:
`UnityEngine.CoreModule` includes them, which we've got through the `Krafs.Rimworld.Ref` NuGet package.